### ISSUE TITLE
The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

### ISSUE THREAD

**Issue by nhutthanh340** [chaquo/chaquopy#526]
#### Describe your issue
I have defined the method void predict_on_batch inside class PredictGPU
![image](https://user-images.githubusercontent.com/36669308/127807223-a874f256-2543-4ac0-9c7a-c9ea1a2ce4b8.png)
I have declared class PredictGPU in python code
![image](https://user-images.githubusercontent.com/36669308/127807444-6a16a5bc-8fec-4cc2-be58-29ff72ea1d62.png)
and call method predict_on_batch 
![image](https://user-images.githubusercontent.com/36669308/127807062-e798850d-96b2-4ca8-a7f7-f2a8fa95df05.png)
When i run app on debug mode, predict_on_batch method is called, but throw error when return.
How can anyone help me, please?

**Comment by nhutthanh340** [comment]
I have tested with inputs=None ==> It working
but when inputs not None then app throw error above despite of nothing in method

**Comment by mhsmith** [comment]
Thanks for the report, we'll fix this in the next version of Chaquopy.

Meanwhile, you can work around the problem by converting the NumPy array to a Java array manually, like this:

```
from java import jarray, jfloat
predict.predict_on_batch(jarray(jarray(jarray(jarray(jfloat))))(data), data_len[0])
```

**Comment by nhutthanh340** [comment]
Thanks, I have resolved it by your

**Comment by mhsmith** [comment]
In case other people have the same problem, let's leave the issue open until it's fixed,

**Comment by mhsmith** [comment]
This issue is fixed in Chaquopy 10.0.1. To upgrade, edit your app's top-level build.gradle file and change the version number of `com.chaquo.python:gradle`.
